File: 1/1000s/15s/1523/CountOddNumbersInAnIntervalRange.py
# https://leetcode.com/problems/count-odd-numbers-in-an-interval-range/?envType=daily-question&envId=2025-12-07

class Solution:
    def countOdds(self, low: int, high: int) -> int:
        resultantCount=0
        if low&1:
            resultantCount+=1
        elif high&1:
            resultantCount+=1
        resultantCount+=((high-low)//2)
        return resultantCount

File: 1/1000s/15s/1523/CountOddNumbersInAnIntervalRange.cpp
/*
https://leetcode.com/problems/count-odd-numbers-in-an-interval-range/?envType=daily-question&envId=2025-12-07
*/

class Solution {
public:
    int countOdds(int low, int high) {
        int res=0;
        if(low&1) res++;
        else if(high&1) res++;
        res+=((high-low)/2);
        return res;
    }
};